src/encoded/schemas/workflow_run.json

```diff
@@ -115,13 +115,20 @@
                         "description": "a specified output file",
                         "type": "string",
                         "linkTo": "File"
+                    },
+                    "value_qc": {
+                        "title": "Output Quality Control",
+                        "description": "a specified output report",
+                        "type": "string",
+                        "linkTo": "QualityMetric"
                     }
                 }
             }
         },
         "output_quality_metrics": {
```

Comment:
Figuring we can patch existing items and then delete this field; _or_ just write an upgrader for it and then delete.
Either can be done in a separate branch.